patches/minecraft/net/minecraft/world/inventory/BrewingStandMenu.java.patch

```diff
@@ -5,7 +5,7 @@
  
        public boolean m_5857_(ItemStack p_39121_) {
 -         return PotionBrewing.m_43506_(p_39121_);
-+         return net.minecraftforge.common.brewing.BrewingRecipeRegistry.isValidIngredient(p_39121_);
++         return true;
```

Comment:
Again, I'm pretty sure this should not just be true always and check the valid potion ingredients. But as for the other datagen pr, it could be argued either way.

Reply:
to check it I would need the level but this is a Slot which has no itea of the level

Reply:
The slot is part of the menu which can have a context of the level, so this is untrue.

Reply:
the slot has no context of level in the mayPlace method

Reply:
I looked within the menu, and it does seem there is nothing for the level. Though most menu containers take in the level, so it could be adapted. But, it doesn't matter either way so just leave it for now.